packages/hawtio/src/plugins/camel/route-diagram/visualization-service.ts

```diff
@@ -234,7 +234,7 @@ class VisualizationService {
         siblingNodes = siblings
       } else {
         siblingNodes = [nodes.length - 1]
-        console.log(siblingNodes, [nodes.length - 1], nodes)
+        siblingNodes = [nodes.length - 1]
```

Comment:
why do we need to repeat the same thing twice?

Reply:
huh 😕  .. we don't thanks!